enums/View.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum View {
+  Table = "table",
+  Kanban = "kanban",
+  Charts = "charts",
```

Comment:
statistics

Reply:
is the second value required?

Reply:
Yes, value is required as there will be some if statements dependent on the View property.